Example: replicated deployments (#1632)
liqotech/liqo: README.md

* [Offloading a Service](https://docs.liqo.io/en/stable/examples/service-offloading.html): learn how to create a multi-cluster service, and how to consume it from each connected cluster.
* [Stateful Applications](https://docs.liqo.io/en/stable/examples/stateful-applications.html): find out how to deploy a database across a multi-cluster environment, leveraging the Liqo storage fabric.
* [Global Ingress](https://docs.liqo.io/en/stable/examples/global-ingress.html): discover how route external traffic to multi-cluster applications through a global ingress and automatic DNS configurations.

> * [Offloading multiple Pods](https://docs.liqo.io/en/stable/examples/offloading-multiple-pods.html): TODO

Reviewer: ```suggestion
* [Replicated Pods](https://docs.liqo.io/en/stable/examples/offloading-multiple-pods.html): learn how to offload an application on multiple remote clusters, replicating its pods on all the available clusters.
```

> * [Replicated Deployments](https://docs.liqo.io/en/stable/examples/replicated-deployments.html): learn how to offload an application on multiple remote clusters, replicating its pods on all the available clusters.

Reviewer 2: Can you add an entry for the terraform example, please? Sorry, we missed it with the right pr

* [Offloading a Service](https://docs.liqo.io/en/stable/examples/service-offloading.html): learn how to create a multi-cluster service, and how to consume it from each connected cluster.
* [Stateful Applications](https://docs.liqo.io/en/stable/examples/stateful-applications.html): find out how to deploy a database across a multi-cluster environment, leveraging the Liqo storage fabric.
* [Global Ingress](https://docs.liqo.io/en/stable/examples/global-ingress.html): discover how route external traffic to multi-cluster applications through a global ingress and automatic DNS configurations.

> * [Replicated Deployments](https://docs.liqo.io/en/stable/examples/replicated-deployments.html): learn how to offload an application on multiple remote clusters, replicating its pods on all the available clusters.

Reviewer: ```suggestion
* [Replicated Deployments](https://docs.liqo.io/en/stable/examples/replicated-deployments.html): learn how to deploy an application by replicating it on multiple remote clusters.
```
